### PR TITLE
Add Amsterdamse sleutel and straatcode to BAG

### PR DESCRIPTION
**Pull request by jasper-jager**

### CODE REVIEW

File: src/tests/enricher/test_enrich_bag.py [hunk 1]
```diff
@@ -25,9 +40,39 @@ def test_enrich_nummeraanduidingen(self):
         self.assertEqual(entities[0]['ligt_in_bag_woonplaats'],'3594')
         self.assertEqual(entities[1]['ligt_in_bag_woonplaats'],'1024')
 
+        self.assertEqual(entities[0]['amsterdamse_sleutel'],'sleutel')
+        self.assertEqual(entities[1]['amsterdamse_sleutel'],'sleutel2')
+
```
Comment (Vegter) [inline src/tests/enricher/test_enrich_bag.py]: What happens if the key cannot be found? Can you add a test for this.

File: src/tests/enricher/test_enrich_bag.py [hunk 2]
```diff
@@ -25,9 +40,39 @@ def test_enrich_nummeraanduidingen(self):
...
+            }
+        ]
+
+        enricher = BAGEnricher("app", "bag", "openbareruimtes")
+        for entity in entities:
+            enricher.enrich(entity)
+
+        # Check if the both sleutel and straatcode have been added
+        self.assertEqual(entities[0]['amsterdamse_sleutel'],'sleutel')
+        self.assertEqual(entities[0]['straatcode'],'code')
```
Comment (Vegter) [inline src/tests/enricher/test_enrich_bag.py]: Also only the success path is tested

File: src/gobimport/enricher/bag.py [hunk 3]
```diff
@@ -29,6 +55,102 @@ def __init__(self, app_name, catalogue_name, entity_name):
...
+
+    def _download_amsterdam_sleutel_file(self, catalogue_name, entity_name):
+        """Download the Amsterdamse sleutel file from the objectstore
+
+        :param catalogue_name:
+        :param entity_name:
+        :return:
+        """
+        connection, user = connect_to_objectstore(get_objectstore_config('Basisinformatie'))
+        container_name = os.getenv("CONTAINER_BASE", "acceptatie")
```
Comment (JKoetsier) [inline src/gobimport/enricher/bag.py]: Maybe move the retrieving and setting of default value of env vars to config.py

File: src/tests/enricher/test_enrich_bag.py (hunk 1 above)
Comment (jasper-jager) [reply src/tests/enricher/test_enrich_bag.py]: Attribute should be empty, added in the test

File: src/tests/enricher/test_enrich_bag.py (hunk 2 above)
Comment (jasper-jager) [reply src/tests/enricher/test_enrich_bag.py]: Attribute should be empty, added in the test

File: src/gobimport/enricher/bag.py (hunk 3 above)
Comment (jasper-jager) [reply src/gobimport/enricher/bag.py]: CONTAINER_BASE is now imported from config.py

**Pull request merged**